packages/@react-aria/virtualizer/src/VirtualizerItem.tsx

```diff
@@ -69,7 +69,7 @@ export function layoutInfoToStyle(layoutInfo: LayoutInfo, dir: Direction, parent
     opacity: layoutInfo.opacity,
     zIndex: layoutInfo.zIndex,
     transform: layoutInfo.transform,
-    contain: layoutInfo.allowOverflow ? 'size layout style' : 'size layout style paint'
+    contain: 'size layout style'
```

Comment:
fixes issue with sticky checkboxes and focus indicator not sticking properly in Safari 15.4. Don't think we need `paint` since we have `overflow` handling that for us in the VirtualizerItem style. Lemme know if you think we should keep it as is, the Safari 15.4 bug is [fixed](https://bugs.webkit.org/show_bug.cgi?id=238016) and may be coming out soonish, but I feel like the above change is still more "correct"